ts/screens/profile/analytics/index.ts

```diff
@@ -12,6 +12,12 @@ export async function trackProfileLoadSuccess(state: GlobalState) {
   await mixpanelTrack(profileLoadSuccess.toString());
 }
 
+export function trackIdpAuthenticationSuccessScreen(idpId: string | undefined) {
+  void mixpanelTrack("IDENTIFICATION_CONFIRM_SCREEN", {
+    idp: idpId,
+    ...buildEventProperties("UX", "screen_view")
+  });
+
```

Comment:
the closing bracket is missing

```suggestion
}
```

Reply:
This was due to the merge conflict resolution. Good catch! 🚀